rpcserver.nim

```diff
@@ -0,0 +1,7 @@
+import
+  eth-rpc / server / servertypes,
```

Comment:
why do avoid using the idiomatic syntax for module imports?

```nim
import
  eth-rpc/server/[servertypes, ehtprocs, etc]
```

Reply:
I had some problems with this actually, which is why it ended up like that.
Using
```
import
  eth-rpc/server/[servertypes, rpcconsts, serverdispatch, ethprocs]
```
I get:
`cannot open file: eth-rpc/server/`, where the error points to the hyphen.
Then a second error, which is probably just as a result of the first error messing up parsing:
`invalid module name: '[servertypes, rpcconsts, serverdispatch, ethprocs]`

I've had a second look, and I can get around the issue by doing this, though it looks kind of odd:
```
import
  r"eth-rpc/server" / [servertypes, rpcconsts, serverdispatch, ethprocs]
```
Seems as if the dash is messing things up, but only if you're using path / [moduleA, moduleB] syntax.

Reply:
I guess the best approach is to avoid dashes in the directory names in general.